hooks/airbrake/airbrake.go

```diff
@@ -25,6 +25,11 @@ func NewHook(endpoint, apiKey, env string) *airbrakeHook {
 }
 
 func (hook *airbrakeHook) Fire(entry *logrus.Entry) error {
+	go hook.fire(entry) // Don't block logging
```

Comment:
I think a better way to do this is that `NewHook` spawns a go routine that reads off a channel, avoiding spawning an unlimited number of goroutines. The channel should be buffered, and if `fire()` can't write to it it prints an error like the current one.